docs/source/data_ingestion_and_preparation/transforms/sketcher.md

```diff
@@ -0,0 +1,12 @@
+## Sketcher
+The Sketcher transform annotates each Document with a sketch that can be used to identify near-duplicate documents.  This process is the prerequisite for later removing or collapsing near-duplicate documents.  Currently, the sketch consists of a set of hash values called `shingles`.  These are relatively inexpensive to calculate and can safely be a default part of any ingestion pipeline.  Using `Sketcher` in a Sycamore data prep pipeline is relatively easy:
```

Comment:
Is the sketch stored as metadata with each Document?

Reply:
Yes, it's metadata, but it's not a property.  Is it clear enough to say "annotates each Document"?  Seems like maybe not.

Reply:
It's clear!